UI/obs-frontend-api/obs-frontend-api.h

```diff
@@ -49,6 +49,8 @@ enum obs_frontend_event {
 	OBS_FRONTEND_EVENT_RECORDING_UNPAUSED,
 
 	OBS_FRONTEND_EVENT_TRANSITION_DURATION_CHANGED,
+	OBS_FRONTEND_EVENT_TBAR_VALUE_CHANGED,
```

Comment:
New events are supposed to be added to the end of the enum.  See https://github.com/obsproject/obs-studio/commit/a38cd03e065efbc863bc897012900938aa70beb3.